### ISSUE TITLE
Deprecated Event

### ISSUE THREAD

**Issue by seema84** [sarhatabaot/ChunkSpawnerLimiter#71]
Paper Version 1.21.1

`[CSL] "ChunkSpawnerLimiter v4.3.10" has registered a listener for com.destroystokyo.paper.event.brigadier.CommandRegisteredEvent on method "public void com.cyprias.chunkspawnerlimiter.libs.acf.PaperBrigadierManager.onCommandRegister(com.destroystokyo.paper.event.brigadier.CommandRegisteredEvent<com.destroystokyo.paper.brigadier.BukkitBrigadierCommandSource>)", but the event is Deprecated. "This event has been superseded by the Commands API and will be removed in a future release. Listen to LifecycleEvents.COMMANDS instead."; please notify the authors [Cyprias, sarhatabaot].`

**Comment by sarhatabaot** [comment]
This isn't really an issue. Waiting on a fix from the commands library devs, this feature will be disabled next release so the error will likely not show up anymore.

**Comment by M1noa** [comment]
There are still deprecation errors.. or there are now atleast..
